projects/plugins/jetpack/extensions/blocks/blogroll/use-subscriptions.js

```diff
@@ -35,6 +36,11 @@ export default function useSubscriptions( { ignore_user_blogs } ) {
 						)
 					);
 				}
+				// Add placeholder image if site icon is missing
+				response = response.map( subscription => {
+					subscription.site_icon = getSiteIcon( subscription.site_icon );
+					return subscription;
+				} );
```

Comment:
I think this is better.
```suggestion
				response = response.map( subscription => {
				        // if the icon is an empty string, return `undefined` so the default attribute kicks in.
					subscription.site_icon = subscription.site_icon || undefined;
					return subscription;
				} );
```

Reply:
Reverted this change. It won't display the place holder in the appender if we dont apply this logic:

```
subscription.site_icon = getSiteIcon( subscription.site_icon );
```